lib/onelogin/ruby-saml/attribute_service.rb

```diff
@@ -14,18 +19,29 @@ def configure(&block)
         instance_eval &block
       end
 
+      # @return [Boolean] True if the AttributeService object has been initialized and set with the required values
+      #                   (has attributes and a name)
       def configured?
         @attributes.length > 0 && !@name.nil?
       end
 
+      # Set a name 
+      # @param name [String] The service name
+      #
       def service_name(name)
         @name = name
       end
 
+      # Set an index 
+      # @param name [Integer] An index
+      #
       def service_index(index)
         @index = index
       end
-      
+
+      # Add an attribute
+      # @param name [Hash] Attribute for the AttributeService
```

Comment:
`@param` should describe `options` and if possible, please provide `@option`s of the possible options that this method accepts